lib/components/Input/input-phone-number/dropdown-content.tsx

```diff
@@ -44,24 +44,77 @@ const DropdownContent = ({
             country.phone_code.includes(searchKey),
     );
 
+    const selectedItemRef = useRef<HTMLLIElement>(null);
+
+    useEffect(() => {
+        if (isOpen && selectedItemRef.current) {
+            setTimeout(() => {
+                const selectedItem = selectedItemRef.current;
+
+                if (selectedItem) {
+                    if (isMobile) {
+                        // Get the search bar height
+                        const searchBar =
+                            document.querySelector(".phone-code-search");
```

Comment:
Can we have a more specific element targeting, ex: the parent container of the exact component. There is a low chance of having thesame class in another component but just to make sure that we are really getting the element from this component.

Reply:
sure let me do that 🙏